plugins/by-name/lazygit/default.nix

```
@@ -63,8 +63,19 @@ lib.nixvim.plugins.mkVimPlugin {
       Config file path is evaluated if this value is `1`.
     '';
 
-    config_file_path = defaultNullOpts.mkNullable (with types; either str (listOf str)) [
-    ] "Custom config file path or list of custom config file paths.";
+    config_file_path =
+      defaultNullOpts.mkNullable
+        (
+          with types;
+          oneOf [
+            str
+            (listOf str)
+            rawLua
+          ]
+        )
+        [
+        ]
```

Does nixfmt allow: 

```suggestion
        [ ]
```
?

---

Yes, but it's ugly. Changed it.